timbermill-java/timbermill-client/src/main/java/com/datorama/oss/timbermill/unit/Event.java

```diff
@@ -322,7 +322,7 @@ public void trimAllStrings() {
 			Map<String, Number> newMetrics = Maps.newHashMap();
 			for (Map.Entry<String, Number> entry : metrics.entrySet()) {
 				Number value = entry.getValue();
-				String key = entry.getKey();
+				String key = entry.getKey().replace(".", "_");
 				if (value != null) {
```

Comment:
actually, I saw we already do that in getTrimmedLongValues to all fields **beside** metric:
		strings = getTrimmedLongValues(strings, Constants.STRING);
		context = getTrimmedLongValues(context, Constants.CTX);
		text = getTrimmedLongValues(text, Constants.TEXT);
		metrics = removeNaNs();
maybe was there a reson?

Reply:
No reason as far as I can see.

Reply:
Let's put it all in one place.

Reply:
hope I understood it right